spec/services/process_submission_service_spec.rb

```diff
@@ -176,15 +198,15 @@
       end
 
       it 'gets the detail_objects from the Submission' do
-        expect(submission).to receive(:detail_objects).at_least(:once).and_return(submission.detail_objects)
+        expect(submission).to receive(:detail_objects).at_least(:once).and_call_original
```

Comment:
this line was hiding the problem from the tests as it stubbed the actual payload